sstables/mp_row_consumer.cc

```diff
@@ -80,9 +81,35 @@ atomic_cell make_counter_cell(api::timestamp_type timestamp, bytes_view value) {
     return ccb.build(timestamp);
 }
 
+// See #6130.
+static data_type freeze_types_in_collections(data_type t) {
+    return ::visit(*t, make_visitor(
+    [] (const map_type_impl& typ) -> data_type {
```

Comment:
typ? type would be more English less Polish I guess :)

Reply:
Same in all the other lambdas here.

Reply:
Typ is an abbreviation for type ;)

Reply:
typical :)